config.py

```diff
@@ -5,43 +5,35 @@
 class Config:    
```

Comment:
Go over the config params, If there is no use in some params -> delete them.
If you use them, but do not pull the value from config file, then use the config file.

Reply:
LOG_Q_VALUE ,'LOG_CAR_STATES', 'EPOCHS' , deleted